build/external-dependencies.props

```diff
@@ -197,6 +197,161 @@ not building again in this patch.
 
   <!-- Shipped dependencies from previous builds -->
 
+  <!-- 2.0.8 -->
+  <ItemGroup>
+    <ExternalDependency Include="Microsoft.AspNetCore.Antiforgery" Version="2.0.3" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.Abstractions" Version="2.0.3" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.Cookies" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.Core" Version="2.0.3" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.Facebook" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.Google" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.MicrosoftAccount" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.OAuth" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.Twitter" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication.WsFederation" Version="2.0.4" Metapackage="false" RuntimeStore="false" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authentication" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authorization.Policy" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Authorization" Version="2.0.4" Metapackage="true" RuntimeStore="true" Category="ship" />
+    <ExternalDependency Include="Microsoft.AspNetCore.Buffering" Version="2.0.3-rtm-10026" Category="noship" />
```

Comment:
So we never build timestamp free versions of the non-shipping packages?

Reply:
nope.